Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _standalone-gateway:
+
+*************************
+Cilium Standalone Gateway
+*************************
+
+Cilium has a standalone mode where it can be operated outside of Kubernetes
+as a standalone gateway. The gateway is designed to reside at the edge of the
+cluster, potentially replacing dedicated hardware components, and handling
+high-traffic load from a north-south direction going into the cluster(s).
+
+The Cilium Standalone Gateway includes functionality such as a high performance
+eBPF-based layer 4 load-balancer and a NAT46/64 gateway. There is a wide range
+of use cases, including:
+
+- Frontend DNS resolver
+- Load-balancer for legacy VM-based workloads,
+- As a gateway in front of IPv6 single stack Kubernetes clusters to allow for
+  interconnection with IPv4,
+- As a load-balancer for kubeapi-server,
```

Comment:
nit, I think this will satisfy the spellchecker too and avoid needing to spellcheck `kubeapi` which is a less common abbreviation:
```suggestion
- As a load-balancer for kube-apiserver,
```

Reply:
Switching to the content, is this a common need to expose kube-apiserver outside the cluster? Seems like something that most users would want to avoid due to security concerns 🤔